components/SpotlightModal/SpotlightModalContent.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FunctionComponent } from 'react';
+
+interface SpotlightModalContentProps {
+  description?: string;
+  source?: string;
+  close: () => void;
+}
+
+const SpotlightModalContent: FunctionComponent<SpotlightModalContentProps> = ({ close, description, source }) => {
+  const renderSourceText = () => {
+    if (source) {
+      return (
+        <>
+          <b className="sourceElement">
```

Comment:
dash case for classnames. Not required here since you're using `<style jsx>`. Doesn't hurt though, so just rename.

Reply:
Updated